### ISSUE TITLE
vk_validation_error_messages.h Missing on Windows build of Android

### ISSUE THREAD

**Issue by sfricke-samsung** [LunarG/VulkanTools#968]
Haven't had much time to look into it so raising the issue here.

Trying to build freshly clone ToT Android builds and it ended up working as normal (getting external sources, generating source, ndk-build) on Linux (Ubuntu 18.04) but on Windows when I try to build for Android I get

```
In file included from jni/../third_party/Vulkan-ValidationLayers/layers/vk_layer_utils.cpp:22:
In file included from jni/../third_party/Vulkan-ValidationLayers/layers/vk_layer_utils.h:33:
jni/../third_party/Vulkan-ValidationLayers/layers/vk_layer_logging.h:50:10: fatal error: 'vk_validation_error_messages.h' file not
      found
#include "vk_validation_error_messages.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

The Linux build has `/build-android/generated/include/vk_validation_error_messages.h` but Windows is missing that file. On Windows I do see it in `VulkanTools\build-android\third_party\Vulkan-ValidationLayers\layers\generated\vk_validation_error_messages.h` though.

**Comment by davidlunarg** [comment]
This problem was introduced when VVL switched from on-the-fly code generation to prebuilt generated files. The Android.mk file needs to change to search for include files in a different place. Currently working on this.

Not sure why Android builds work on Linux but not on Windows.

**Comment by davidlunarg** [comment]
Fixed in commit a056c9acd1a5243eb5a9d63fd119f5155ad7ed5e. Closing.
